components/Header/index.js

```diff
@@ -0,0 +1,9 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return <StyledHeader>Hammerhart</StyledHeader>;
+}
+
+const StyledHeader = styled.h1`
```

Comment:
only a suggestion: you could choose a consisent naming-convention including "Styled"-prefix for a styled Component or without it.